frame/pablo/src/liquidity_bootstrapping.rs

```diff
@@ -203,7 +203,6 @@ impl<T: Config> LiquidityBootstrapping<T> {
 		let base_amount = repatriate(pool.pair.base)?;
 		let quote_amount = repatriate(pool.pair.quote)?;
 
-		Pools::<T>::remove(pool_id);
```

Comment:
what is reason to terain pool  if there is no liquidity?

Reply:
we may want some basic details about pool. like in this PR, remove_liquidity() triggers TWAPUpdated event and for that I need pool's asset ids. if I allow pool to be deleted here I need to emit event from this specialized part of remove_liquidity() call. 
IMO we it would be better to have a separate extrinsic to delete pool.

Reply:
i have seen that one extinsic can do 2 events. its ok.

Reply:
may be ok if some ED is retained on pool account

Reply:
I will more Pools::<T>::remove() call to generic remove_liquidity() call where LBP is handled.